src/worknet/Node/WorkNetNode.cs

```diff
@@ -7,255 +8,256 @@
 using Neo.BlockchainToolkit.Plugins;
 using Neo.Cryptography;
 using Neo.IO;
+using Neo.Json;
 using Neo.Network.P2P.Payloads;
 using Neo.Network.RPC;
 using Neo.Persistence;
 using Neo.Plugins;
 using Neo.SmartContract;
+using Neo.SmartContract.Manifest;
 using Neo.SmartContract.Native;
 using Neo.Wallets;
-
 using NeoArray = Neo.VM.Types.Array;
 using NeoStruct = Neo.VM.Types.Struct;
 
 namespace NeoWorkNet.Node;
 
 class WorkNetNode
 {
-    readonly WorknetChain chain;
-    readonly string dataDirectory;
-
-    public WorkNetNode(WorknetChain chain, string filename)
+  readonly WorknetChain chain;
```

Comment:
Why did the spacing change from 4 spaces to 2? Can we standardize on 4 spaces please?